build.gradle

```diff
@@ -18,7 +18,7 @@ buildscript {
 plugins {
 	id 'java'
 	id 'idea'
-	id 'org.springframework.boot' version '2.7.4'
+	id 'org.springframework.boot' version '2.7.18'
```

Comment:
Will org.springframework.boot:spring-boot-gradle-plugin and org.springframework.boot always have the same version? If so consider a var to store version, otherwise, this is fine.

Reply:
I'm going to leave it separate for now.